### PR TITLE
docs(demos): add dayjs demo

### PR DESCRIPTION
**Pull request by xigolle**
Integrates the updated `angular-utils` version with dayjs support in `angular-calander`.
Continues on the great work that has been done in this [PR](https://github.com/mattlewis92/calendar-utils/pull/43).
Should fix issue #1387. 

@mattlewis92 I tried my best to implement it but please double check as this is my first time contributing to an open source project. 
So if there are things that are wrong let me know!

### CODE REVIEW

File: projects/angular-calendar/src/date-adapters/dayjs/index.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { adapterFactory as baseAdapterFactory } from 'calendar-utils/date-adapters/moment';
```
Comment (mattlewis92) [inline projects/angular-calendar/src/date-adapters/dayjs/index.ts:1]: I don't think we even need a specific adapter for dayjs do we? As the API is identical to moment's we can just use that one and keep it simple?

File: README.md [hunk 2]
```diff
@@ -179,6 +179,8 @@ https://github.com/mattlewis92/angular-bootstrap-calendar
 
 Run `npm start` to start a development server on port 8000 with auto reload + tests.
 
+> To run this on a windows machine change `npm start` in `package.json` from `node --max_old_space_size=8192 ./node_modules/.bin/ng serve --open` to `node --max_old_space_size=8192 ./node_modules/@angular/cli/bin/ng serve --open`.
```
Comment (mattlewis92) [inline README.md:182]: Sorry for the friction here! I just pushed [a commit](https://github.com/mattlewis92/angular-calendar/commit/f5b50e87bc0e980887c268cc015abdd6a9666856) that should get rid of this issue, can you let me know if it works ok for you?

**Review by mattlewis92 — CHANGES_REQUESTED** [review]
This is great work, thank you so much! Just one suggestion where we can make things a bit simpler by removing the extra date adapter and then we should be good to land this! 😄

File: README.md (hunk 2 above)
Comment (xigolle) [reply README.md]: True. I added it as it looked like the convention but indeed DayJs uses the same API as Moment for those functions. 
Maybe if this changes in the future we can readd it.

File: projects/angular-calendar/src/date-adapters/dayjs/index.ts (hunk 1 above)
Comment (xigolle) [reply projects/angular-calendar/src/date-adapters/dayjs/index.ts]: True. I added it as it looked like the convention but indeed DayJs uses the same API as Moment for those functions.
Maybe if this changes in the future we can readd it.

File: README.md (hunk 2 above)
Comment (xigolle) [reply README.md]: Worked like a charm! Removed the lines from the doc.

**Comment by xigolle** [comment]
@mattlewis92 I think I did the changes you requested but the PR doesn't seem to want to merge automatically. 
So I think I screwed up somewhere or is it something you can fix on your side ? :)

**Review by mattlewis92 — APPROVED** [review]
Great work, thank you!!

**Pull request merged**